### ISSUE TITLE
fit one cycle 

### ISSUE THREAD

**Issue by asalahDyab** [fastai/fastai2#423]
I am new in deep learning. any one could tell me the function and purpose of freeze function. and I am wondering if 1 epoch its enough because if I increase the number of epochs the accuracy will be lower.

![image](https://user-images.githubusercontent.com/36843337/88282627-facf4780-ccfa-11ea-858b-5687295831a4.png)

**Comment by swayson** [comment]
The freeze function, will make some layers of weights not change during training. This is useful in transfer learning, where you have pre-trained a model but now want to transfer what it learned to a new task, but not forget what it has learned. Freezing layers gives a control of how rapidly that takes place. Generally, a process of gradual unfreezing is applied, where systematically more and more layers are 'unfrozen' and so their weights will start to change during training, but perhaps at lower learning rates so it doesn't forget what it learned during pre-training.

This is covered in quite a lot of detail in the fast.ai video lessons.
